resources/templates/base.html

```diff
@@ -29,9 +29,27 @@ <h1 class="display-3">GraphViz Pocket Reference</h1>
         </button>
         <div class="collapse navbar-collapse" id="main-navbar">
             <ul class="navbar-nav mr-auto">
-                {% for navs in main_nav %}
-                    <li class="nav-item"><a class="nav-link" href="{{ navs.url }}" class="{{ navs.classes }}">{{ navs.title }}</a></li>
-                {% endfor %}
+                <li class="nav-item">
```

Comment:
Since there was no breadcrumbs in a long time, moved from generating this list to just hardcoding it